tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -433,3 +434,47 @@ def test_rjumps_jumpf_nonreturning(
         possible_exceptions.append(EOFException.INVALID_NON_RETURNING_FLAG)
 
     eof_test(container=Container(sections=sections), expect_exception=possible_exceptions or None)
+
+
+def gen_stack_underflow_params():
+    """Generate parameters for stack underflow tests."""
+    for op in sorted(valid_eof_opcodes):
+        if op.popped_stack_items == 0:
```

Comment:
We should be able to use `min_stack_height` instead because it normally is equal to `popped_stack_items`, but in some cases (like for `DUP*` and `SWAP*`) `popped_stack_items == 0` while `min_stack_height > 0`.